SUMMARY.md

```diff
@@ -99,6 +99,13 @@
         * [Import HTML composition](guides/web-apps/import-html-composition/README.md)
         * [Avoiding URI conflicts](guides/blending/avoiding-uri-conflicts/README.md)
         * [Avoiding CSS conflicts](guides/blending/avoiding-css-conflicts/README.md)
+    * [Publishing in the App Warehouse](guides/publishing/README.md)
```

Comment:
This link won't work if you don't put the `publishing/README.md` in the guides directory.